data-prepper-plugins/grok-processor/src/main/resources/grok-patterns/patterns

```diff
@@ -0,0 +1,15 @@
+ALB_URI: (%{WORD:protocol})?(://)?(%{IPORHOST:domain})?(:)?(%{INT:http_port})?(%{GREEDYDATA:request_uri})?
```

Comment:
We should have unit tests for these log patterns. At the very least, let's have a unit test that verifies that the patterns are loaded and available for use, perhaps by creating a new Grok processor with one of these and ensuring that it validates.

Reply:
I'll add these patterns to `GrokProcessorIT` to at least use them in match to make sure they are available to use. Validating all of them in integration test input and output could be nice as well, but I doubt it would be maintained as more patterns are added. I'll see what I can do for that